TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift

```diff
@@ -30,6 +30,7 @@ struct PushFactor: Factor, Codable {
   let createdAt: Date
   let config: Config
   var keyPairAlias: String?
+  var metadata: [String: String]? = nil
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/8493b62acdbfcf89b7484fe450452cd8fa9157ed/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift#L33"></span>Initializing an optional variable with nil is redundant.<br />
<code>redundant_optional_initialization</code> <code>PushFactor.swift:33</code></td>
    </tr>
  </tbody>
</table>